### ISSUE TITLE
Docker deployment map is gray

### ISSUE THREAD

**Issue by brodyFlannigan** [ibi-group/datatools-ui#983]
## Observed behavior (please include a screenshot if possible)

When using the docker quickstart (which is amazing, by the way) The map is simply gray, without any map background. Stop locations are visible but there's nothing behind them. This is an issue even with a mapbox key set in the configuration. Looking at the network tab, I'm seeing requests being made to `http://localhost:9966/feed/40a70eaf-9bc1-402b-9867-604a479d12b5/optional-map-tile-url` when I move the map around, which doesn't seem to be normal.

![image](https://github.com/ibi-group/datatools-ui/assets/46612183/c4d80b40-9c75-4eae-94d9-e0adebb0cc80)

## Expected behavior

Map should have background
## Steps to reproduce the problem
1. Download from commit `6fc72d2148e8dda005aae9c2b26ba911e4197506`
2. In the appropriate location (/configurations/default/env.yml), insert your mapbox API key.
3. Run `docker-compose up` as suggested in README.

## Any special notes on configuration used
Nothing special other than changing my mapbox key in /configurations/default/env.yml
## Version of datatools-ui and datatools-server if applicable (exact commit hash or branch name)

Commit `6fc72d2148e8dda005aae9c2b26ba911e4197506`

**Comment by philip-cline** [comment]
Hi @brodyFlannigan, thanks for reporting this! We'll take a look and get back to you shortly

**Comment by miles-grant-ibigroup** [comment]
Thanks for reporting! We'll be able to patch this soon. As a stopgap fix, you can open `configurations/env.yml.tmp` (and `configurations/env.yml` if you have one) and remove line 5 (the one with "optional-map-tile-url"). If it's removed datatools should fall back to a default that works!

We'll put out a patch that removes this unhelpful configuration line.

**Comment by brodyFlannigan** [comment]
The removal of line 5 (Optional map tile URL) did the trick! Thanks!

I'll leave the issue open as I understand you're working on a more long-term solution :)

**Comment by ArthurSrz** [comment]
I did remove line 5 but it did not work. My map stays grey. Did you setup a more permanent fix ? Thanks !

**Comment by miles-grant-ibigroup** [comment]
Does your network inspector show any failed requests? If so, could you please share a screenshot?

**Comment by ArthurSrz** [comment]
Nope. No failed requests. There seems to be no requests made to mapbox...

**Comment by miles-grant-ibigroup** [comment]
We found a fix for this! Please add `MAP_BASE_URL: http://tile.openstreetmap.org/{z}/{x}/{y}.png` to your front-end config

**Comment by TheChilliPL** [comment]
This fix doesn't seem to be working for me. The MAP_BASE_URL was commented, however uncommenting fixed nothing, just like making and setting my own Mapbox API key. The map is still all gray

**Comment by miles-grant-ibigroup** [comment]
> This fix doesn't seem to be working for me. The MAP_BASE_URL was commented, however uncommenting fixed nothing, just like making and setting my own Mapbox API key. The map is still all gray

Let's chat more in the gitter! There's a chance you're not putting the config in the right place
